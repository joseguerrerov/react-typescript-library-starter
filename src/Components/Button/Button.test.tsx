import React from 'react';
import { shallow } from 'enzyme';
import { Button } from './index';

describe('Button Component', () => {
  describe('Render', () => {
    it('renders', () => {
      const wrapper = shallow((
        <Button onClick={() => null}>
          Test
        </Button>
      ));
      expect(wrapper).toMatchSnapshot();
    });
  });
});
