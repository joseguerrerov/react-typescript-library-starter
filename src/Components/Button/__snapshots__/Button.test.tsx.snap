// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Render renders 1`] = `
<BaseButton
  onClick={[Function]}
>
  Test
</BaseButton>
`;
